src/modules/UI/scenes/SendConfirmation/SendConfirmation.ui.js

```diff
@@ -275,7 +276,7 @@ export class SendConfirmation extends Component<Props, State> {
   }
 
   networkFeeSyntax = () => {
-    const { networkFee, parentNetworkFee, parentDisplayDenomination } = this.props
+    const { networkFee, parentNetworkFee, parentDisplayDenomination, exchangeRates } = this.props
     if (!networkFee && !parentNetworkFee) return ''
```

Comment:
See if we can just use `fiatPerCrypto` rather than `exchangeRates`

Reply:
I am going to leave this as is because the logic within the render function is a bit too involved for me to decipher at this time